java/serverless_lab1/src/main/java/com/ibmdevelopersf/serverless_lab1/ServerlessLab1.java

```diff
@@ -0,0 +1,26 @@
+package com.ibmdevelopersf.serverless_lab1;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+
+import java.util.logging.Logger;
+
+public class ServerlessLab1 {
+
+    protected static final Logger logger = Logger.getLogger("ServerlessLab1");
+
+    public static JsonObject main(JsonObject args) {
+        JsonObject response = new JsonObject();
+        JsonPrimitive nameArg = args.getAsJsonPrimitive("name");
+        String result;
+        if (nameArg == null) {
+            result = "Hello World!";
+        } else {
+            result = "Hello " + nameArg.getAsString();
+        }
+        response.addProperty("greeting", result);
```

Comment:
@blumareks I do have quotes. I forgot to add in the comments :).

Reply:
https://github.com/IBMDeveloperSF/serverless-labs/pull/7/files#diff-3358af545e312f75e2e5c326de5f1024R21